packages/contracts/contracts/protocol/libraries/types/DataTypes.sol

```diff
@@ -47,17 +47,17 @@ library DataTypes {
     }
 
     enum ReserveAssetType {
-        AAVE, //0
+        CHAINLINK, //0
         CURVE, //1
         CURVEV2, //2
         YEARN, //3
         BEEFY, //4
         VELODROME, //5
         BEETHOVEN, //6
-        RETH //7
+        RETH, //7
+        CL_PRICE_ADAPTER //8
```

Comment:
I took a look at assetmappings `setAssetType` and its not really being used? shouldn't something in the setup script call setAssetType to set the correct oracle for the BASE assets (or any assets)

Reply:
It is set in `addAssetMapping` dw